web/docs/deployment/deployment-methods/wasp-deploy/railway.md

```diff
@@ -96,6 +96,20 @@ wasp deploy railway setup <project-name>
 wasp deploy railway deploy <project-name>
 ```
 
+#### Using a custom database image
+
+Use `--db-image <docker-image>` to provision the database with a specific Postgres image (for example, to enable extensions):
+
+```shell
+# PostGIS
+wasp deploy railway launch my-wasp-app --db-image postgis/postgis
+
+# pgvector
+wasp deploy railway launch my-wasp-app --db-image pgvector/pgvector:pg16
+```
```

Comment:
I would turn these into two shell code blocks, since you don't want to run one and then the other.